lib/bugsnag/resque.rb

```diff
@@ -37,7 +37,7 @@ def save
 # Auto-load the failure backend
 Bugsnag::Resque.add_failure_backend
 
-Resque.before_first_fork do
+Resque.after_fork do
   Bugsnag.configuration.app_type = "resque"
   Bugsnag.configuration.delivery_method = :synchronous
```

Comment:
In Shopify we have a custom delivery method that we use for all the types of workers (web and jobs). We are aware that using async delivery method may not work for jobs, but not being possible for us to use our custom delivery method in jobs is blocking us to upgrade this gem.